src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -224,6 +224,12 @@ float_array_map prepare_batch(std::vector<st_example>& batch, size_t width,
       shared_float_array::wrap(std::move(index_array), {batch_size});
   map.emplace("index", index_wrap);
 
+  shared_float_array image_width_wrap = shared_float_array::wrap(width);
+  map.emplace("width", image_width_wrap);
```

Comment:
This is fine, but FYI I think this should also be perfectly acceptable:
```
map["width"] = shared_float_array::wrap(width);
```
Technically this form might be slightly less efficient, since it default constructs the value for the key before overwriting it with the RHS, but generally I err on the side of clarity and simple code